cmd/kube-controller-manager/app/options/options.go

```diff
@@ -92,6 +92,8 @@ func NewCMServer() *CMServer {
 			LeaderElection:          leaderelection.DefaultLeaderElectionConfiguration(),
 			ControllerStartInterval: unversioned.Duration{Duration: 0 * time.Second},
 			EnableGarbageCollector:  false,
+			ClusterSigningCertFile:  "/var/lib/kubernetes/ca/ca.pem",
+			ClusterSigningKeyFile:   "/var/lib/kubernetes/ca/ca.key",
```

Comment:
We have used /srv/kubernetes, /var/lib/kubelet, and /etc/kubernetes but never this. @roberthbailey what is the convention?